server/backup_runner.go

```diff
@@ -155,7 +156,9 @@ func (r *BackupRunner) purgeBinlogFiles(instanceID, retentionPeriodTs int) error
 	}
 	for _, binlogFileInfo := range binlogFileInfoList {
 		if _, err := mysql.GetBinlogNameSeq(binlogFileInfo.Name()); err != nil {
-			log.Warn("Found an irregular file in binlog directory.", zap.String("path", binlogFileInfo.Name()))
+			if binlogFileInfo.Name() != "metadata" {
```

Comment:
the binlog file name is something like binlog.000080.meta, how does it compare with `metadata`

Reply:
This is a directory. Per discussion with danny [here](https://github.com/bytebase/bytebase/pull/2330#discussion_r950817423), I'll put the .meta files in the same directory with binlog files. This loc will be gone.